RecoMuon/L3MuonProducer/test/newL3.py

```diff
@@ -202,31 +202,26 @@ def producers_by_type(process, type):
 	
 	
 	#OI Seeding:
-	process.hltIterL3OISeedsFromL2Muons = cms.EDProducer("TSGForOI",
-	        MeasurementTrackerEvent = cms.InputTag("hltSiStripClusters"),
-	        UseHitSeeds = cms.bool(True),
-	        adjustErrorsDynamicallyForHitless = cms.bool(True),
-	        adjustErrorsDynamicallyForHits = cms.bool(True),
-	        debug = cms.untracked.bool(False),
-	        estimator = cms.string('hltESPChi2MeasurementEstimator100'),
-	        fixedErrorRescaleFactorForHitless = cms.double(5.0),
-	        fixedErrorRescaleFactorForHits = cms.double(2.0),
-		pT1 = cms.double(13.0),				       
-		pT2 = cms.double(30.0),				       
-		pT3 = cms.double(70.0),				       
-		eta1 = cms.double(1.0),				       
-		eta2 = cms.double(1.4),				       
-                SF1 = cms.double(3.0),
-                SF2 = cms.double(4.0),
-                SF3 = cms.double(5.0),
-                SF4 = cms.double(7.0),
-                SF5 = cms.double(10.0),
-		hitsToTry = cms.int32(5),
-	        layersToTry = cms.int32(3),
-	        maxEtaForTOB = cms.double(1.1),
-	        maxSeeds = cms.uint32(1),
-	        minEtaForTEC = cms.double(0.9),
-	        src = cms.InputTag("hltL2Muons","UpdatedAtVtx")
+    process.hltIterL3OISeedsFromL2Muons = cms.EDProducer("TSGForOIDNN",
```

Comment:
This indentation is incorrect. The file is actually broken in the first place (try `python3 newL3.py`), b/c of mixed use of tabs and whitespaces. Please fix it by removing tabs, and reformatting (or, remove the file altogether if the MUO POG prefers that).

Reply:
I have fixed the indentation problem, now it should run correctly. For the tabs/whitespaces part, I will wait to see if we want to remove the file or keep it.

Reply:
Hi @andrea21z, this customizer is obsolete and does not compatible with the Run 3 menu. So I suggest you remove the file.

Reply:
I agree, I will remove the test file.